_drafts/2020-01-01-umd-cs-vs-nyu-cs-pros-and-cons-a-transfer-perspective.md

```diff
@@ -0,0 +1,77 @@
+
+# Social Life
+I'm not an extrovert. I'm not really an introvert either. I'm not sure which category I fit in. I do know that I don't enjoy the social life at UMD, which consists of football games and frat parties. I enjoy a wholesome gathering or checking out restaurants. UMD's college town, College Park, isn't really know for great restaurants if you cannot tell. Washington D.C doesn't really have a food scene either since it consists mostly of goverment buildings. Gentrification has increased the rent prices in D.C's [Chinatown](https://www.washingtonpost.com/lifestyle/style/dcs-chinatown-has-only-300-chinese-americans-left--and-fighting-to-stay/2015/07/16/86d54e84-2191-11e5-bf41-c23f5d3face1_story.html). The social life at UMD just wasn't for me.
+
+All of this contributes to my transfer decision. Even though I enjoyed the CS courses, where I built a solid foundation, I don't feel that I fit in at UMD as a person or as an international student. However, having experienced a competitive department, I try my best take advatage of the resources at NYU to succeed as a CS student. There are certain aspects I really enjoy at NYU.
+
+# CS education
+NYU CS curriculum is lackluster. The department doesn't even require students to take Calculus II[^1], probability/stats, and programming languages. However, if you are familiar with a decent CS curriculum, you can plan your to take courses with more challenging professor at NYU.
+
+I enjoy the class size at NYU since each class usually has 30-80 students. It allows me to attend office hours more easily and get to know my professors on a personal level. Before the pandemic, sometimes I would go to office hours without having specific questions in mind and discuss different topics with them. Everyone should take avantage of the professors at NYU. They are top of their fields and really knowledgeable. I find that professors here tend to be on the friendlier side. Some even have lots of industry connections so definitely attend their office hours to know them better. They might also be the [founder](https://cs.nyu.edu/~korth/) of [hackNY](https://hackny.org/)! or [cryptographic algorithm inventor](https://en.wikipedia.org/wiki/Cramer%E2%80%93Shoup_cryptosystem). 
```

Comment:
avantage -> advantage

Reply:
I enjoy is a little awkward here. "I really enjoy" for some reason sounds better. Idk why

Reply:
"They are top of their fields" -> "They are at the top of their fields"